src/components/Support/Support.jsx

```diff
@@ -19,17 +27,37 @@ const ranks = {
   }
 };
 
+function formatMoney(number) {
+  let str = Math.round(number) + '';
+  if (str.length > 3) {
+    str = str.substr(0, str.length - 3) + ',' + str.substr(-3);
+  }
+  return str;
+}
+
 export default class Support extends React.Component {
   render() {
-    let { rank, type } = this.props;
-    let supporters = require(`./support-${type}.json`);
+    let { rank } = this.props;
+    let supporters = [
+      ...GoldSponsors,
+      ...SilverSponsors,
+      ...Sponsors,
+      ...Backers,
+    ];
```

Comment:
Since we end up combining anything anyway, maybe we can simplify the fetch script to put everything in one file in the first place, making some of the code here unnecessary. Anyway, I was in a bit of rush right now so I'll review the code more in depth later on and see if there's anything we can simplify. Let me know what you think about the one file idea.